sale_order_product_recommendation_secondary_unit/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Sale Order Product Recommendation Secondary Unit",
     "summary": "Add secondary unit to recommend products wizard",
-    "version": "13.0.2.2.0",
+    "version": "14.0.2.2.0",
```

Comment:
always reset the semver part as doing migration / adding new module. This case it should be `14.0.1.0.0`